pyscriptjs/tests/integration/test_py_runtime_config.py

```diff
@@ -0,0 +1,68 @@
+import os
+import tarfile
+import tempfile
+
+import pytest
+import requests
+
+from .support import PyScriptTest
+
+URL = "https://github.com/pyodide/pyodide/releases/download/0.20.0/pyodide-build-0.20.0.tar.bz2"
```

Comment:
Doesn't need to be in this PR but I'd rather have a list of Pyodide versions and we have a parameterized tests that tests for all of them..

Reply:
oh, it's not about testing `0.20.0`. The context is NOT testing a different version.
The context is testing a local pyodide (instead of hosted one) -- without the `indexURL` stuff -- which was creating a havoc before when it was used earlier.
We are just using a different version for another sanity check, that we load the correct and intended one, instead of the default hosted one.

Reply:
That's fine if the context here is to test a local instance vs hosted. Still, we'd benefit a lot from testing different version of pyodide and understanding if/what is broken in specific versions. (we can add this tests as follow up work)

Reply:
aah ok

Reply:
@fpliger here Madhur has followed a suggestion of mine. The problem was to find a way to check that we are actually using the custom version of pyodide, and the easiest thing to do was to use a different version to have a visible difference.
Eventually this test might become problematic if 0.20.0 will be unsupported, but if/when it happens, we can just use a newer version and/or find a different way to fix the test.

@madhur-tandon that's why adding comments explaining the rationale is a good thing ;)

Reply:
Yes, I have added comments already @antocuni

Reply:
yes sorry, for some reason github didn't update the page and when I wrote my comment I didn't see that the conversation already move forward.